src/main/sensors/barometer.c

```diff
@@ -250,15 +250,15 @@ bool baroDetect(baroDev_t *dev, baroSensor_e baroHardwareToUse)
         }
 #endif
         FALLTHROUGH;
-	
-	 case BARO_QMP6988:
+    
```

Comment:
Replacing tabs with spaces is good. But in the case of extra tabs it is better to remove them altogether. :grinning:

Reply:
Done